resources/examples/manager/BasicAssetLibrary/tests/fixtures.py

```diff
@@ -0,0 +1,105 @@
+#
+#   Copyright 2013-2022 The Foundry Visionmongers Ltd
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+Manager test harness fixtures for the apiComplianceSuite
+"""
+
+import json
+import os
+
+
+from openassetio.constants import kField_EntityReferencesMatchPrefix
+
+#
+# Test library
+#
+
+test_library_path = os.path.join(os.path.dirname(__file__), "resources", "library_apiComplianceSuite.json")
+test_libray = {}
+
+with open(test_library_path, "r", encoding="utf-8") as file:
+    test_library = json.load(file)
+
+#
+# Constants
+#
+
+identifier = "org.openassetio.examples.manager.bal"
+
+valid_ref = "bal:///references/can/contain/🐠"
+non_ref = "not a Ŕeference"
+
+an_existing_ref = next(iter(test_library["entities"].keys()))
+
+
+fixtures = {
+    "identifier": identifier,
+    "settings": {
+        "library_path": test_library_path
+    },
+    "shared": {
+        "a_valid_entity_reference": valid_ref,
+        "a_non_entity_reference": non_ref,
```

Comment:
I struggled with this one. I had thought of invalid too, but wondered if this could be construed as "an entity reference that is not valid" ie. a correctly formatted string, that does not convey valid data. Rather than "a string that is definitely not formatted as an entity reference". How about:

```
"an_entity_reference"
"not_an_entity_reference"
```

Reply:
Hmm, good point. My concern with `a_non_entity_reference` was that, at a glance, I read it to mean "a valid reference to something that is not an entity". How about a `a_malformed_entity_reference`?

I think `not_an_entity_reference` could still fall into the trap of assuming a valid entity reference to an invalid entity, but is still better than `a_non_entity_reference`.  

I kinda like that we could establish a convention of `a_<happy path fixture>` side-by-side with `not_a_<happy path fixture>`, but I suspect, like this case, it would usually be too ambiguous.

Reply:
Does dropping the word "entity" help disambiguate? `a_valid_reference` / `a_malformed_reference`?